lib/include/oclero/qlementine/style/Theme.hpp

```diff
@@ -199,6 +201,7 @@ class Theme {
   QPalette palette;
 
 public:
+  void setUseSystemFont(bool use);
```

Comment:
Could you please remove this? Currently, there is no setter methods in `Theme`. The `bool` can be changed, but it's up to the caller to call `initializeFonts()` after modifying the `Theme`.